docs/configs/html/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = u'Daml SDK'
-copyright = u'© Copyright 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved. Any unauthorized use, duplication or distribution is strictly prohibited. "Digital Asset" and "Daml" are Registered in the U.S. Patent and Trademark Office.'
+copyright = u'2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved. Any unauthorized use, duplication or distribution is strictly prohibited. "Digital Asset" and "Daml" are Registered in the U.S. Patent and Trademark Office.'
```

Comment:
Any chance you could change that on the template side instead? Having a different string just for this one copyright notice across the entire repo means it will get forgotten every year when we need to update it.

Reply:
Good point. FIxed.